### PR TITLE
CLUE LEDs (White and Red)

### PR DESCRIPTION
**Pull request by vandyliu**
[AB#34437](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34437)
With @xnkevinnguyen 
# Description:
- Added functionality for White LEDs (two on front) and Red LED (one on back).
On:
LEDs on
Off:
LEDs off

**Testing code for blinking red leds and white*
```
from adafruit_clue import clue
import time


temp = True
while True:

    clue.white_leds = temp
    clue.red_led = temp
    time.sleep(0.1)
    temp = not temp

```
## Type of change

Please delete options that are not relevant.
- [ ] New feature (non-breaking change which adds functionality)

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/view/components/clue/Clue_svg.tsx [hunk 1]
```diff
@@ -2,22 +2,36 @@
...
+import svg from "../cpx/Svg_utils";
 export interface IRefObject {
     [key: string]: React.RefObject<SVGRectElement>;
 }
 interface IProps {
     displayImage: string;
-    neopixel: number[];
+    leds: {
+        neopixel: number[];
+        redLed: boolean;
```
Comment (isadorasophia) [inline src/view/components/clue/Clue_svg.tsx]: nit: I would rename for isRedLedOn? Or something no these lines (same for the white led)

File: src/view/components/cpx/Svg_utils.tsx [hunk 2]
```diff
@@ -77,6 +77,22 @@ namespace svg {
         el.appendChild(t);
         return t;
     }
+
+    export function setLed(
+        ledStatus: boolean,
```
Comment (isadorasophia) [inline src/view/components/cpx/Svg_utils.tsx:82]: nit: I would also rename this for something like isLedOn

File: src/view/components/clue/Clue_svg.tsx [hunk 3]
```diff
@@ -1051,29 +1133,59 @@ export class ClueSvg extends React.Component<IProps, {}> {
...
-            this.pixelStopGradient.current.setAttribute("stop-color", rgbColor);
+            this.gradientRefs.neopixel.current.setAttribute(
+                "stop-color",
+                rgbColor
+            );
+        }
+    }
+    private updateLeds() {
+        // update white led
+        const { whiteLed, redLed } = this.props.leds;
```
Comment (isadorasophia) [inline src/view/components/clue/Clue_svg.tsx]: nit: also isWhiteLedOn, isRedLedOn

**Review by isadorasophia — APPROVED** [review]
Cool! :D Thanks for the test script.

**Review by andreamah — APPROVED** [review]
Looks great! 💖

**Pull request merged**